extension.js

```diff
@@ -1020,7 +1050,7 @@ class ClipboardIndicator extends PanelMenu.Button {
   }
 
   _deleteEntryAndRestoreLatest(entry) {
-    this._removeEntry(entry, true, true);
+    this._confirmRemoveEntry(entry, true, true);
```

Comment:
Not correct either because this shouldn't trigger a confirmation: https://github.com/SUPERCILEX/gnome-clipboard-history/blob/d437412d4584c4299f06bbe4d74322f35a695302/extension.js#L914

I think you need to take in an optional confirm param.